charts/azuremonitor-containers/templates/ama-logs-daemonset.yaml

```diff
@@ -83,6 +83,11 @@ spec:
          valueFrom:
             fieldRef:
               fieldPath: status.hostIP
+       - name: CONTAINER_MEMORY_LIMIT_IN_BYTES
```

Comment:
lets get this manifest update as part of this week's AKS RP release, so that we can get the agent changes through toggle first to see if this change working as expected and without any side effects.

Reply:
acknowledged